tests/arp/test_arp_dualtor.py

```diff
@@ -66,10 +66,14 @@ def garp_setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, config_facts
     clear_dut_arp_cache(duthost)
 
     vlan_intfs = config_facts['VLAN_INTERFACE'].keys()
-    garp_enable_cmd = 'redis-cli -n 4 HSET "VLAN_INTERFACE|{}" grat_arp enabled'
+    garp_check_cmd = 'sonic-db-cli CONFIG_DB HGET "VLAN_INTERFACE|{}" grat_arp'
+    garp_enable_cmd = 'sonic-db-cli CONFIG_DB HSET "VLAN_INTERFACE|{}" grat_arp enabled'
     cat_arp_accept_cmd = 'cat /proc/sys/net/ipv4/conf/{}/arp_accept'
     arp_accept_vals = []
+    old_grat_arp_vals = {}
     for vlan in vlan_intfs:
+        old_grat_arp_res = duthost.shell(garp_check_cmd.format(vlan))
+        old_grat_arp_vals[vlan] = old_grat_arp_res['stdout']
```

Comment:
Is there a case where there is no grat_arp keyword? In that case the value statement will trigger exception.

Reply:
If there is no such field, the output will simply be an empty string (this will apply to all single ToR testbeds). In this case the fixture is still able to setup/teardown properly

Reply:
Will you fail the test at line 99 below?